ampproject/amphtml
# 🐛 [amp-video-docking] Fix issue hiding docked player during scroll on mobile

Test-Case:

- Start via `amp --extensions=amp-video-iframe,amp-video-docking`
- Open http://localhost:8000/examples/article-with-docked-video.html with the iOS simulator
- Play first video in article
- Scroll down so that the player is docked
- Now scroll further down and then quickly up so that the URL bar is shown again
- Result => dock gets hidden

This pull request fixes the problem by only checking whether the width changed on resize to decide whether the player should be undocked.

Also see this comment, which also explains the bug: https://github.com/ampproject/amphtml/issues/27825#issuecomment-656565478

cc @alanorozco
